src/WorkflowManager/WorkflowExecuter/Common/ArtifactMapper.cs

```diff
@@ -76,6 +78,18 @@ IStorageService storageService
             return artifactPathDictionary;
         }
 
+        private static string? GetArtifactSuffix(string valueString)
+        {
+            var variableStrings = valueString.Split("}");
+
+            if (variableStrings.Length < 2)
+            {
+                return null;
+            }
+
+            return variableStrings[1];
```

Comment:
I'm trying to think is it possible to have multiple variables in this field?
might be worth doing variableStrings.Last() I think thats linq way of getting last element.